src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -13610,17 +13629,26 @@ public void Field_Event_Type_Update_RuntimeTypeUnchanged(string oldType, string
         [Theory]
         [InlineData("int", "string")]
         [InlineData("int", "int?")]
```

Comment:
>int [](http://example.com/codeflow?start=21&length=3)

Why not test the tuple type?

Reply:
Because the tests have to make sure they're finding the right symbols, since both can exist in a compilation, its hard to use theories. I thought I'd added a standalone test for the tuple case, but I can't see it, so I will do so.